File: C++/Manual 01/Question02.cpp
#include <iostream>
using namespace std;

int main()
{
    float weight, height, bmi;

    // Input height in inches and weight in pounds
    cout << "Input your height in inches: ";
    cin >> height;
    cout << "Input your weight in pounds: ";
    cin >> weight;

    // Calculate BMI
    bmi = (weight * 703) / (height * height);

    // Display BMI category
    if (bmi >= 18.5 && bmi <= 25)
    {
        cout << "Your BMI is optimal" << endl;
    }
    else if (bmi < 18.5)
    {
        cout << "You are underweight!" << endl;
    }
    else
    {
        cout << "You are overweight!" << endl;
    }

    return 0;
}

File: JavaScript/Manual 01/Question02.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter your Weight in Pounds: ", function (weight) {
    rl.question("Enter your Height in Inches: ", function (height) {
        let bmi = (weight / (height * height)) * 703;

        if (bmi < 18.5) {
            console.log("Underweight");
        } else if (bmi >= 18.5 && bmi <= 24.9) {
            console.log("Normal");
        } else if (bmi >= 25 && bmi <= 29.9) {
            console.log("Overweight");
        } else {
            console.log("Obese");
        }

        rl.close();
    });
});